### PR TITLE
[Feat] Keyboard navigation

### PR DESCRIPTION
**Pull request by MartnLecam**
# Name of the feature/issue
Keyboard navigation

## Description
Add logic to use hotkey hook which will allow to navigate the extension with hotkeys. It is implemented in header so no need to include it on each page

## Type of change

Click the correct/s option/s

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  This change requires a documentation update

### CODE REVIEW

File: packages/ui/src/util/hooks/useHotKey.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            )
+
+            if (navigateTo) {
+                if (typeof navigateTo === "string") {
+                    console.log("NavigateTo")
+                    history.push({
+                        pathname: navigateTo,
+                        state: {
+                            from: history.location.pathname,
+                            //Acordarse revisar el estado del state
```
Comment (fpezzola) [inline packages/ui/src/util/hooks/useHotKey.ts:63]: What about this?

**Review by fpezzola — COMMENTED** [review]
This pr is pretty good 👏 

But it is full of `console.log` for debugging porposes. Can you get rid of them after converting this PR ready to review? 

Also, can we standardize the hotkeys definisions "alt+n" in a constants/utils file? It could be in the `utils/hotkeys` file.

**Comment by fpezzola** [comment]
> This pr is pretty good 👏
> 
> But it is full of `console.log` for debugging porposes. Can you get rid of them after converting this PR ready to review?
> 
> Also, can we standardize the hotkeys definisions "alt+n" in a constants/utils file? It could be in the `utils/hotkeys` file.

Added to this comment, I tested the hotkeys on mac and they are not working as expected. Also, the documentation (modal) is not including the keys for macOS

**Comment by MartnLecam** [comment]
###Important
After development is approved. We will check the design for hotkeys screen and define a Wiki also.

**Comment by MartnLecam** [comment]
Close this PR to merge with extension redesign

**Pull request closed without merging**